services/checks/checks.go

```diff
@@ -346,6 +363,9 @@ func (s *Service) executeMySQLChecks(ctx context.Context) ([]string, error) {
 				s.l.Warnf("Failed to process action result: %s.", err)
 				continue
 			}
+			// record metrics
+			s.mExecutedScripts.WithLabelValues(string(models.MySQLServiceType)).Inc()
```

Comment:
Do we need an explicit `string` conversion there? (I did not check, but probably it will work without explicit conversion)

Reply:
I tried that first but they editor showed me red squiggly lines

```
cannot use c.Type (variable of type check.Type) as string value in argument to s.mGeneratedAlerts.WithLabelValues
```

I was surprised that it can't infer that `Type` is actually a string